style.css

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+  font-size: 100%;
+}
+
+/* NAVIGATION BAR */
+
+.navbar {
+  margin: 46px 6.5% 0 6.5%;
+}
```

Comment:
-  [OPTIONAL] Great job implementing the changes requested by the previous reviewer here. :+1: :+1: I noticed that the margin-top is slightly higher than the one in the given design. I'll suggest you reduce the margin to match the given Figma design. changing it to this works fine `.navbar {
  margin: 8px 6.5% 0 6.5%;
}`

- SCREENSHOT REFERENCE OF THE EXTRA MAGIN-TOP
![Selection_015](https://user-images.githubusercontent.com/107632780/234949566-e54ed147-d8aa-4c1f-9b5f-ed7ea1c89e7b.png)

Reply:
Hi Valentine! Many thanks for your feedback!